blocks/trending-news/trending-news.css

```diff
@@ -0,0 +1,153 @@
+.trending-news-container {
```

Comment:
I'd suggest making a section variant like `shadow` and not have section level styles at block level.

Reply:
done